provisioning/RHCSA_RHCE_LAB/handlers/main.yml

```diff
@@ -2,10 +2,8 @@
 - name: update pip
   shell: pip install --upgrade pip
 
-- name: Upgrade packages
-  yum:
-    name="*"
-    state="latest"
```

Comment:
Maybe I took this out because it took so long :-)

Soon we'll have RHEL 7.3 although it might take some time for these to show up in the exams. e.g. RHEL 7.0 uses [`nmcli` version without `ipv4.gateway`](https://github.com/smartbit/labipa-rhcsa-rhce)